File: app/src/main/java/com/example/moviehub/MainActivity.java

package com.androidglide;


import android.os.Bundle;
import android.view.View;
import android.widget.Button;
import android.widget.ImageView;

import androidx.appcompat.app.AppCompatActivity;

import com.bumptech.glide.Glide;

public class MainActivity extends AppCompatActivity {
    String IMAGE_URL = "https://www.thecrazyprogrammer.com/wp-content/uploads/2015/09/Neeraj-Mishra.png";
    Button button;
    ImageView image;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        val articleAdapter = ArticleAdapter(this, articles)
        articlesRecyclerView.adapter = articleAdapter
        setContentView(R.layout.activity_main);

        button = (Button)findViewById(R.id.button);
        image = (ImageView)findViewById(R.id.image);

        //load image on button click
        button.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View v) {
                Glide.with(MainActivity.this).load(IMAGE_URL).into(image);
            }
        });
    }
}

File: app/src/main/java/com/example/moviehub/MainActivity.kt
package com.example.moviehub

import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import androidx.recyclerview.widget.LinearLayoutManager
import androidx.recyclerview.widget.RecyclerView
import com.codepath.asynchttpclient.AsyncHttpClient
import com.codepath.asynchttpclient.RequestParams
import com.example.moviehub.RequestHandler
import com.example.moviehub.http.RequestHandler
import com.example.moviehub.model.results.PersonResults
import com.example.moviehub.rView.PersonAdapter
import com.google.gson.Gson

class MainActivity : AppCompatActivity() {
    private val gson = Gson()
    private val client = AsyncHttpClient()
    private val params = RequestParams()

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)

        // person logic
        val personRV = findViewById<RecyclerView>(R.id.trending_people_RV)
        RequestHandler.getTrendingPeople(personRV)

        // movie logic
        val movieRV = findViewById<RecyclerView>(R.id.trending_movie_RV)
        LinearLayoutManager(this).also { movieRV.layoutManager = it }
        RequestHandler.getTrendingMovies(movieRV)
    }

    private fun LinearLayoutManager(mainActivity: MainActivity): LinearLayoutManager {

    }
}